opus_functions.cmake

```diff
@@ -202,6 +202,12 @@ function(opus_supports_cpu_detection RUNTIME_CPU_CAPABILITY_DETECTION)
   endif()
   if(HAVE_INTRIN_H OR HAVE_CPUID_H)
     set(RUNTIME_CPU_CAPABILITY_DETECTION 1 PARENT_SCOPE)
+  elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "(armv7-a|aarch64)")
+    # ARM cpu detection is implemented for Windows and anything
+    # using a Linux kernel (such as Android).
+    if (CMAKE_SYSTEM_NAME MATCHES "(Windows|Linux|Android)")
```

Comment:
lets add iOS as well

Reply:
https://cmake.org/cmake/help/latest/variable/IOS.html

Reply:
The reason I didn't add it here was that it doesn't appear that runtime capability detection is implemented for iOS.